charts/karpenter/values.yaml

```diff
@@ -143,7 +143,7 @@ controller:
   sidecarVolumeMounts: []
   metrics:
     # -- The container port to use for metrics.
-    port: 8080
```

Comment:
Did this change or was this just completely wrong?

Reply:
I was forced to change this due to the knative hardcoded port value. See the previous PR description.

Reply:
Ah, I missed that. I assume that this value is propagated into our serviceMonitor. I think this is technically a breaking change for those that have manually created their service monitors or have other logic. Should we document this in our upgrade guide?

Reply:
> Metrics are still scrapable at the karpenter service on port 8080

Reading the PR description, I don't get this line

Reply:
svc/karpenter forwards to the pod on a different port: 8080 -> 8000. Compatible with scrapers. 
```
k port-forward svc/karpenter -n karpenter 8080
Forwarding from 127.0.0.1:8080 -> 8000
Forwarding from [::1]:8080 -> 8000
Handling connection for 8080
```

Reply:
In the service monitor, it's discovered by name:
```
  endpoints:
    - port: http-metrics
      path: /metrics
```

Reply:
Got it, so it really only affects users who are referring to the port by name. Makes sense.